WorkflowSwiftUI/Tests/Derived/ObservableStateTests.swift

```diff
@@ -35,23 +36,29 @@ final class ObservableStateTests: XCTestCase {
         withPerceptionTracking {
             _ = state.child
         } onChange: {
-            XCTFail("state.child should not change.")
+            if tracker.isComplete == false {
+                XCTFail("state.child should not change.")
+            }
```

Comment:
This is the same kind of approach used in Perception now: https://github.com/pointfreeco/swift-perception/blob/main/Tests/PerceptionTests/PerceptionTrackingTests.swift#L32

The issue is that the registrar deinit pathway will now trigger an onChange which we don't really want to "count" in this context.

Related to https://github.com/pointfreeco/swift-perception/discussions/153

<img width="1834" height="770" alt="image" src="https://github.com/user-attachments/assets/87718dce-6ba5-48d8-abfd-18ce6c569cc7" />